pkg/storage/segments/store_test.go

```diff
@@ -484,6 +274,50 @@ func TestCalcNeededNodes(t *testing.T) {
 			Total:            tt.n,
 		}
 
-		assert.Equal(t, tt.needed, calcNeededNodes(&rs), tag)
+		assert.Equal(t, tt.needed, segments.CalcNeededNodes(&rs), tag)
+	}
+}
+
+func createTestData(t *testing.T, size int64) []byte {
+	data, err := ioutil.ReadAll(io.LimitReader(rand.Reader, size))
+	require.NoError(t, err)
+	return data
+}
+
+func createSegmentStore(t *testing.T, planet *testplanet.Planet) (segments.Store, metainfo.Client) {
+	// TODO move apikey creation to testplanet
+	project, err := planet.Satellites[0].DB.Console().Projects().Insert(context.Background(), &console.Project{
+		Name: "testProject",
+	})
+	require.NoError(t, err)
+
+	apiKey := console.APIKey{}
+	apiKeyInfo := console.APIKeyInfo{
+		ProjectID: project.ID,
+		Name:      "testKey",
 	}
+
+	// add api key to db
+	_, err = planet.Satellites[0].DB.Console().APIKeys().Create(context.Background(), apiKey, apiKeyInfo)
+	require.NoError(t, err)
+
+	TestAPIKey := apiKey.String()
+
+	oc, err := planet.Uplinks[0].DialOverlay(planet.Satellites[0])
+	require.NoError(t, err)
+
+	metainfo, err := planet.Uplinks[0].DialMetainfo(context.Background(), planet.Satellites[0], TestAPIKey)
+	require.NoError(t, err)
+
+	ec := ecclient.NewClient(planet.Uplinks[0].Transport, 0)
+	fc, err := infectious.NewFEC(4, 6)
+	require.NoError(t, err)
+
+	rs, err := eestream.NewRedundancyStrategy(eestream.NewRSScheme(fc, 1*memory.KB.Int()), 4, 6)
```

Comment:
Any specific reason to set the `repairThreshold` to the `requiredCount` in the `NewRedundancyStrategy` constructor. If you don't care, just pass `0` for both `repairThreshold` and `optimalThreshold`.